scitt/statement_creation.py

```diff
@@ -0,0 +1,175 @@
+"""The issuer creates the statement and signs it.
```

Comment:
Will restore this soon. The registration script servces as a full example of receipt verification for now